crates/lang/ir/src/ir/item_impl/message.rs

```diff
@@ -100,6 +100,8 @@ pub struct Message {
     pub(super) item: syn::ImplItemMethod,
     /// If the ink! message can receive funds.
     is_payable: bool,
+    /// If the ink! message is a wildcard selector.
+    is_wildcard_selector: bool,
```

Comment:
yes please ... ^^

Reply:
Yeah, so I had originally considered this. But it requires this:
```rust
pub struct Selector {
    bytes: [u8; 4],
}
```
to be made into
```rust
pub struct Selector {
    bytes: Option<[u8; 4]>,
}

fn is_wildcard(&self) -> bool {
  self.bytes.is_none()
}
```
This is no trivial change though, as it requires a lot of new checks throughout our codebase and more special logic, whereas with the way I implemented it currently it's way less of a diff.

Reply:
I think this will be very easy if we instead make a new type that wraps the `ir::Selector`:
```rust
pub enum SelectorOrWildcard {
    Wildcard,
    Selector(ir::Selector),
}
```

Reply:
Done.